scripts/plugins/support-pssl.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+#
+# Copyright (c) Citrix Systems 2017. All rights reserved.
+
+set -e
+
+# Open/Close port 6640 when pssl is used in ovs
+
+
+OP="$1"
+CHAIN="RH-Firewall-1-INPUT"
+POSITION=$(iptables -L --line-n | grep -A 11 "Chain $CHAIN" | grep "https" | cut -d " " -f1)
```

Comment:
Can we do "iptables -L --line-n -n" and check for port 443 (instead of https), we dont want iptables to do address resolution.